docs/src/manual/arrays.md

```diff
@@ -4,89 +4,81 @@ Symbolic array-valued expressions (symbolic arrays) are supported by Symbolics.
 
 You can create a symbolic array variable with the following syntax:
 
-```julia
+```@example arrays
+using Symbolics
 @variables A[1:5, 1:3] b[1:3]
 ```
 
 Here `A` is a symbolic matrix of size `(5, 3)` and `b` is a symbolic vector of length 3.
 
-```julia
-julia> size(A)
-(5, 3)
-
-julia> size(b)
-(3,)
-
-julia> ndims(A)
-2
-
-julia> ndims(b)
-1
-
-julia> eltype(A)
-Real
-
-julia> eltype(b)
-Real
+```@example arrays
+size(A)
+```
+```@example arrays
+size(b)
+```
+```@example arrays
+ndims(A)
+```
+```@example arrays
+ndims(b)
+```
+```@example arrays
+eltype(A)
+```
+```@example arrays
+eltype(b)
```

Comment:
Instead of manually writing `@example` so many times, you could use [`@repl`](https://juliadocs.github.io/Documenter.jl/v0.27/man/syntax/#@repl-block).

Reply:
Personally, I prefer not to have the repl prompt in there.
But I'll change it to whatever the maintainers prefer.

Reply:
I also prefer to not have the REPL prompt.